LIBITD-2539. Add Test for check-view

aatishgupta25 opened the pull request:
Added tests/test_api_check.py to test the logic of the CheckView and also checked values from the reponse provided.

https://umd-dit.atlassian.net/browse/LIBITD-2539

Commit:
LIBITD-2539. Add Test for check-view

Added tests/test_api_check.py to test the logic of the
CheckView and also checked values from the reponse provided.

https://umd-dit.atlassian.net/browse/LIBITD-2539

Commit:
LIBITD-2539. Added one more assert

Added another assert to check for the f-string returned
in the JSON for an unparseable ip address.

https://umd-dit.atlassian.net/browse/LIBITD-2539

peichman-umd commented on tests/test_api_check.py:
```diff
@@ -0,0 +1,38 @@
+
+  # Checking if the response has the group key and ip_address that was passed 
+  # in as query parameter
+  assert response.json()['group']['key']=='A'
```
When you are using the JSON data multiple times in the same function, I would suggest saving it to a variable to avoid re-parsing it every time. For small data structures we have here it isn't really an issue, but it is a good habit to get into.

```python
response_data = response.json()
assert response_data['group']['key'] == 'A'
...
```

peichman-umd reviewed (changes requested):
Logic is good, and tests all pass. Just one code suggestion.

Commit:
LIBITD-2539. Modified test_api_check.py

Store response.json() in a local object and used it
instead of re-parsing each time

https://umd-dit.atlassian.net/browse/LIBITD-2539
